zendesk/zendesk.go

```diff
@@ -273,3 +275,24 @@ func (z *Client) includeHeaders(req *http.Request) {
 		req.Header.Set(key, value)
 	}
 }
+
+// addOptions build query string
+func addOptions(s string, opts interface{}) (string, error) {
+	v := reflect.ValueOf(opts)
```

Comment:
is there a way to do this while avoiding adding reflection? 

From the go proverbs:
```
Clear is better than clever.
Reflection is never clear.
```

https://go-proverbs.github.io/

Reply:
ok, nil check at caller enables to remove this reflection, right?

Reply:
Fixed at 335f8af